lib/extensions/devise_async_email/devise/models/authenticatable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Extensions::DeviseAsyncEmail::Devise::Models::Authenticatable
+  def self.included(base)
+    base.prepend(AsyncEmail)
+  end
+
+  module AsyncEmail
```

Comment:
You can just call this PrependMethods, and remove the self.included hook altogether, I believe.